components/cards/ParcelInfo.tsx

```diff
@@ -224,52 +219,42 @@ function ParcelInfo(props: ParcelInfoProps) {
   }, [licenseDiamondAddress, sfFramework, paymentToken]);
 
   React.useEffect(() => {
-    (async () => {
-      if (ceramic == null || !ceramic.did) {
-        console.error("Ceramic instance not found");
-        return;
-      }
-
-      setAssetContentManager(null);
+    if (!licenseAddress || !data?.geoWebParcel || !selectedParcelId) {
+      return;
+    }
 
-      if (selectedParcelId && licenseOwner) {
-        const assetId = new AssetId({
-          chainId: `eip155:${NETWORK_ID}`,
-          assetName: {
-            namespace: "erc721",
-            reference: registryContract.address.toLowerCase(),
-          },
-          tokenId: new BN(selectedParcelId.slice(2), "hex").toString(10),
-        });
+    (async () => {
+      const web3Storage = new Web3Storage({
+        token: process.env.NEXT_PUBLIC_WEB3_STORAGE_TOKEN ?? "",
+        endpoint: new URL("https://api.web3.storage"),
+      });
+      const geoWebContent = new GeoWebContent({
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        ceramic: ceramic as any,
+        ipfs,
+        web3Storage,
+      });
 
-        const accountId = new AccountId({
-          chainId: `eip155:${NETWORK_ID}`,
-          address: licenseOwner,
-        });
+      setGeoWebContent(geoWebContent);
 
-        const model = new DataModel({
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          ceramic: ceramic as any,
-          aliases: GeoWebModel,
-        });
+      const assetId = new AssetId({
+        chainId: `eip155:${NETWORK_ID}`,
+        assetName: {
+          namespace: "erc721",
+          reference: registryContract.address.toLowerCase(),
+        },
+        tokenId: new BN(selectedParcelId.slice(2), "hex").toString(10),
+      });
 
-        const _assetContentManager = new AssetContentManager(
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          ceramic as any,
-          model,
-          `did:pkh:${accountId.toString()}`,
-          assetId
-        );
-        setAssetContentManager(_assetContentManager);
-
-        const doc = await _assetContentManager.getIndex();
-        setParcelIndexStreamId(doc.id.toString());
-      } else {
-        setAssetContentManager(null);
-        setParcelIndexStreamId(null);
-      }
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const doc = await TileDocument.deterministic(ceramic as any, {
```

Comment:
Instead of linking to Cerscan to show the Ceramic document, we could link to show the IPLD DAG here https://explore.ipld.io which would avoid needing to calculate the deterministic document here.

@gravenp Do you think we should link to the Ceramic doc or the root IPLD DAG?

Reply:
> @gravenp Do you think we should link to the Ceramic doc or the root IPLD DAG?

Yeah, that could be more practical because Cerscan isn't the most user-friendly for traversing the stream to see the content anyway.

Reply:
If we make that change, we should probably rename the field too right? "Root CID" like it is now on the updated browser?